makoctl

```diff
@@ -4,19 +4,21 @@ usage() {
 	echo "Usage: makoctl <command> [options...]"
 	echo ""
 	echo "Commands:"
-	echo "  dismiss [-a|--all] Dismiss the last or all notifications"
-	echo "  invoke [action]    Invoke an action on the last notification"
-	echo "  list               List notifications"
-	echo "  reload             Reload the configuration file"
-	echo "  help               Show this help"
+	echo "  dismiss [-a|--all]    Dismiss the last or all notifications"
+	echo "  invoke [action] [id]  Invoke an action on the notification"
```

Comment:
Maybe we should use an option for this, having multiple optional arguments is a little bit hairy.

```
invoke [-n id] [action]
```

Thoughts?

Reply:
I like this idea.